sig-scalability/2020-annual-report.md

```diff
@@ -0,0 +1,119 @@
+# 2020 - SIG Scalability - Community Meeting Annual Report
+
+## Operational
+
+**How are you doing with operational tasks in sig-governance.md?**
+
+- **Is your README accurate? have a CONTRIBUTING.md file?**
+
+The README is accurate an up-to-date.
+We don't have CONTRIBUTING.md and we're planning to update our
+[developer documentation]. The [desired documentation] is mentioned
+in the corresponding bug.
+
+[developer documentation]:https://github.com/kubernetes/community/tree/master/contributors/devel/sig-scalability
+[desired documentation]:https://github.com/kubernetes/community/issues/5236#issuecomment-805607895
+
+- **All subprojects correctly mapped and listed in sigs.yaml?**
+
+Yes.
+
+- **What’s your meeting culture? Large/small, active/quiet, learnings?
+Meeting notes up to date? Are you keeping recordings up to date/trends
+in community members watching recordings?**
+
+Quite small (usually 5-10 people) and generally quiet. We try to keep
+meeting notes up to date. We're often lagging with updating recordings :(
+
+**How does the group get updates, reports, or feedback from subprojects?
+Are there any springing up or being retired? Are OWNERS files up to date
+in these areas?**
+
+There isn't any formal process. Updates are given during regular SIG meetings.
+
+**When was your last monthly community-wide update? (provide link to deck and/or recording)**
+
+January'20: [slide deck]
+
+[slide deck]:https://docs.google.com/presentation/d/1M81X3_SWwHrJaWdJsecJReOVNxjjWSBFBkhPKEV35hs/edit#slide=id.g401c104a3c_0_0
+
+
+## Membership
+
+**Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?**
+
+Yes
+
+**How do you measure membership? By mailing list members, OWNERs, or something else?**
+
+Active participants in SIG meetings and/or sending/reviewing PRs.
+TBD
+
+**How does the group measure reviewer and approver bandwidth?
+Do you need help in any area now? What are you doing about it?**
+
+We're not doing a very good here. It happens that some PRs are waiting for weeks.
```

Comment:
One thing that might help would be a general review of the current [subprojects + associated OWNERS files](https://github.com/kubernetes/community/tree/master/sig-scalability) to make sure they're up to date + potentially promote more active folk into the reviewers.

Reply:
Subprojects definitions are up-to-date (I read that when creating this report).

That said - the adding more reviewers (especially for the perf-tests repo) is a good suggestion. I added it to the agenda of the next SIG meeting (next week).